packages/ui/origin-ui-core/src/components/certificates/certificateMenuCreator.tsx

```diff
@@ -64,6 +64,13 @@ function ApprovedCertificationRequestsTable() {
     return <CertificationRequestsTable approved={true} />;
 }
 
+function EnchangeInboxPage(): JSX.Element {
+    return <ExchangeInbox mode={CertificateSource.Exchange} />;
+}
+function BlockchainInboxPage(): JSX.Element {
```

Comment:
Maybe it is better to remove these 2 components from `menuCreator` and make them separate components. 
Including Table specific functions like `claim`, `deposit` for each of them and passing these functions as props to general inbox component.
By doing this you will make Inbox general component more lightweight, easy to read and more reusable.